packages/components/ng-ovh-responsive-popover/package.json

```diff
@@ -38,8 +38,7 @@
     "set-value": "^2.0.1"
   },
   "dependencies": {
-    "@ovh-ux/translate-async-loader": "^1.0.8",
-    "jquery": "^2.1.3"
+    "@ovh-ux/translate-async-loader": "^1.0.8"
```

Comment:
I notice that yarn.lock is not updated , jquery used elsewhere ?

Reply:
Yes ! jQuery still used by other packages